tests/resources/issue.md

```diff
@@ -29,3 +29,7 @@ _No response_
 ### Tags
 
 tool, tutorial, paper implementation
+
+### Website
+
+_No response_
```

Comment:
maybe test with a website?

Reply:
Good idea; done in [490158d](https://github.com/qiskit-community/ecosystem/pull/413/commits/490158d302e8f903933bc9688a9616d72f18885b)